source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -394,6 +394,14 @@ def get_setting(self, name: str) -> Any:
         """
         return self._settings.get(name)
 
+    def update_fabric_and_kinematics(self) -> None:
```

Comment:
I am wondering if we should go for a more canonical terminology. Many simulators distinguish between kinematic and dynamics stepping through different terms. For example, Mujoco calls forward kinematics function as `forward` and the simulation step as `simulate`.
 
Something like this is more user-friendly.

```suggestion
    def forward(self) -> None:
```